public/js/lib/devtools/client/shared/components/reps/window.js

```diff
@@ -37,8 +37,8 @@ define(function (require, exports, module) {
       return "";
     },
 
-    getLocation: function (grip) {
-      return getURLDisplayString(grip.preview.url);
+    getDisplayValue: function (grip) {
+      return grip.preview.isGlobal ? "Global" : "Window";
```

Comment:
Thanks @janodvarko. 

I think updating the *tiny* mode makes a lot of sense. We're already forcing our grip's to be [tiny](https://github.com/devtools-html/debugger.html/blob/master/public/js/components/ObjectInspector.js#L136).

The motivation behind formatting the window object with `Window` or `Global` is in part space constraints and in part it's clearer, in the same way that labeling Arguments as opposed to referring to it as Object. The discussion is here #884.